pnpm-lock.yaml

```diff
@@ -17490,6 +17520,12 @@ packages:
       yargs-parser: 21.1.1
     dev: true
 
+  /yazl/2.5.1:
```

Comment:
It is currently being used in the @sap/ux-ui5-tooling (https://www.npmjs.com/package/@sap/ux-ui5-tooling?activeTab=dependencies) and since one of the main use cases for this module is to be integrated there, I thought it makes sense to use the same.

Reply:
OK. Thanks for the background.